File: wms/src/main/webapp/system/app/model/bdef/bdef_WmsDefstrategyModel.js
/**
 * 模块名称：策略配置-头档
 * 模块编码：I101
 * @author hcx 20160610
 */
Ext.define('cms.model.bdef.bdef_WmsDefstrategyModel',{
	extend:'Ext.data.Model',
	idProperty:'strategyType,strategyId',
	fields:[
        {name:'strategyType'},
        {name:'strategyId'},
        {name:'defaultFlag'},
        {name:'strategyName'},
        {name:'rgstName'},
        {name:'rgstDate'},
        {name:'defaultFlagText'}
        
	]
});

File: model/src/main/java/com/sealinkin/bdef/model/Bdef_WmsDefstrategyModel.java
package com.sealinkin.bdef.model;

import com.sealinkin.bdef.po.Bdef_WmsDefstrategy;

public class Bdef_WmsDefstrategyModel extends Bdef_WmsDefstrategy{

	private static final long serialVersionUID = 1L;
	private String strategyType;
	private Double strategyId;
	private String defaultFlagText;
	
	public String getStrategyType() {
		return strategyType;
	}
	public void setStrategyType(String strategyType) {
		this.strategyType = strategyType;
	}
	public Double getStrategyId() {
		return strategyId;
	}
	public void setStrategyId(Double strategyId) {
		this.strategyId = strategyId;
	}
	public String getDefaultFlagText() {
		return defaultFlagText;
	}
	public void setDefaultFlagText(String defaultFlagText) {
		this.defaultFlagText = defaultFlagText;
	}
	
}
